x-pack/plugins/security_solution/public/common/lib/telemetry/events/insights/index.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { TelemetryEvent } from '../..';
+import { TelemetryEventTypes } from '../..';
+
+export const insightsGeneratedEvent: TelemetryEvent = {
```

Comment:
Thanks for plumbing through the `insightsGenerated` telemetry! 🙂

Reply:
![image](https://media1.giphy.com/media/v1.Y2lkPTc5MGI3NjExdDMxamdlNzc3YW41MHFtMnk0ZW02dDY5d3N1NDI1eHYwcnl1N3B3MyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/HOsHtiVdeypFxOhLAf/giphy.gif)